src/app/components/Tag/Tag.module.css

```diff
@@ -0,0 +1,21 @@
+.tag {
+  font-weight: 700;
+  border: 1px solid var(--color-primary);
+  background: transparent;
+  border-radius: 0.25rem;
+  color: var(--color-primary);
+  padding: 0.6rem 1rem;
+}
+
+.active {
+  border: none;
```

Comment:
The missing pixel makes it move and be less wide than inactive. I'd just add a transparent border of 1px
![image](https://user-images.githubusercontent.com/48555130/133092899-da3288ab-74dc-4bbb-82df-caff82281398.png)
![image](https://user-images.githubusercontent.com/48555130/133092962-3ee747b6-1eff-411e-92f7-70206dc61a6e.png)